turtlebot3c-gadget/recovery.md

```diff
@@ -0,0 +1,109 @@
+# The recovery partition
```

Comment:
Does this belongs here since it's a generic information about ubuntu core image?